src/workerd/io/worker.h

```diff
@@ -638,10 +638,18 @@ class Worker::Actor final: public kj::Refcounted {
 
   using Id = kj::OneOf<kj::Own<ActorIdFactory::ActorId>, kj::String>;
 
+  class Loopback {
+    // Class that allows sending requests to this actor, recreating it as needed. It is safe to hold
+    // onto this for longer than a Worker::Actor is alive.
+  public:
+    virtual kj::Own<WorkerInterface> getWorker(IoChannelFactory::SubrequestMetadata metadata) = 0;
```

Comment:
I assume based on the semantics of this class that it is also safe to hold onto to the `kj::Own<WorkerInterface>` here longer than the actor is alive? If so, it would likely be worthwhile adding a comment to that effect so that it's more obvious. Also, is it ok to hold onto the `kj::Own<WorkerInterface>` longer than the `kj::Own<Loopback>` is alive?